app/utils/chart-config/api.ts

```diff
@@ -11,6 +11,7 @@ export const createConfig = async (state: ConfiguratorStatePublishing) => {
     method: "POST",
     data: {
       data: {
+        version: state.version,
         dataSet: state.dataSet,
         dataSource: state.dataSource,
         meta: state.meta,
```

Comment:
How come the version was not saved ? How could migration work then ?

Reply:
Previously we only had versions of chart config, and not configurator state – now we also keep versions of it, so we have two migrations (config state + chart configs).